### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by coder12git** [community/community#62513] [GitHub Education]
### Select Topic Area

Question

### Body

Hey, my GitHub student developer pack had expired and now when I am trying to reverify my details to use it again, it's not working and asking me to "submit a new request to apply for a discount", although  for creating a new request i need another mail which I have added to my GitHub account, but still I am not able to click on continue. Please check it and let me know as soon as possible.

My GitHub username -> coder12git

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @coder12git, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Awrsha** [comment]
Here are a few things you can try to reactivate your GitHub Student Developer Pack:

Make sure you're using the same email address that you originally used to sign up for the pack. GitHub may be unable to link your new email to your existing account.

Resubmit your student information:

Go to your account settings and click "Developer settings"
Under "Student Developer Pack" click "Resubmit your student information"
Provide updated proof of enrollment like a new student ID or tuition invoice.
If the "Submit new request" button is still not working, try clearing your browser cache and cookies. This has sometimes helped resolve similar issues.

As a last resort, contact GitHub Support directly for assistance:

Explain that your Student Pack has expired and you're unable to reverify your details to reactivate it.

Mention that the "Submit new request" button is not working, even after clearing your cache.

GitHub Support should be able to manually reactivate your Student Pack on their end, or provide more troubleshooting steps.

The key will likely be resubmitting valid student information through the correct process, whether that's the online form working properly or GitHub Support doing it manually.
